src/backend/libc/io/epoll.rs

```diff
@@ -337,24 +264,20 @@ struct Event {
     // the full union; `Context` implementations will simply need to deal with
     // casting the value into and out of the `u64` themselves.
     event_flags: EventFlags,
-    encoded: u64,
+    data: u64,
```

Comment:
Could we make `Event` public now and remove `EventVec` entirely, and have users use `Vec<Event>`?

Reply:
I don't think we can, since `Event` is `repr(packed)` and that's hard to deal with in safe Rust.

Reply:
Ah, makes sense.